### DISCUSSION TITLE
Missing wiki repo in Azure DevOps

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#35] [Comments]
# Missing wiki repo in Azure DevOps

Missing wiki repo in Azure DevOps

https://devopsjournal.io/blog/2020/07/14/Missing-wiki-repo-in-Azure-DevOps

**Comment by jnikolaj** [comment]
Thanks a lot, i was looking arouns for 30min where the branch i created thought VS Code was, apparently it was no where at all.
Lucikly i saw your post and i were succesfully able to find it, and create a pull request into Master because of it.

**Reply by rajbos** [reply]
Thanks for letting me know @jnikolaj!

**Comment by johannesprinz** [comment]
Thank you for the great article. There are some great features in the OOTB wiki, like fixing links when you move and rename pages, I'll see if I can use your information above to see if I can get that to apply to a wiki that generated code first.

**Comment by f-steff** [comment]
Thank you for this writeup!

5 years old, and still better than the official documentation!
